services/housetables/src/test/java/com/linkedin/openhouse/housetables/e2e/togglerule/ToggleStatusesServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.openhouse.housetables.e2e.togglerule;
+
+import com.linkedin.openhouse.common.api.spec.ToggleStatusEnum;
+import com.linkedin.openhouse.common.test.cluster.PropertyOverrideContextInitializer;
+import com.linkedin.openhouse.housetables.services.ToggleStatusesService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.jdbc.Sql;
+
+@SpringBootTest
+@ContextConfiguration(initializers = PropertyOverrideContextInitializer.class)
+@Sql({"/schema.sql", "/data.sql"})
+public class ToggleStatusesServiceTest {
+  @Autowired ToggleStatusesService toggleStatusesService;
+
+  @Test
+  public void testActivatedTableForDummy() {
+    Assertions.assertEquals(
+        toggleStatusesService.getTableToggleStatus("demo", "demodb", "demotable").getStatus(),
+        ToggleStatusEnum.ACTIVE);
+    Assertions.assertEquals(
+        toggleStatusesService.getTableToggleStatus("dummy1", "db", "tbl").getStatus(),
+        ToggleStatusEnum.ACTIVE);
+    Assertions.assertEquals(
+        toggleStatusesService.getTableToggleStatus("dummy1", "db", "testtbl1").getStatus(),
+        ToggleStatusEnum.ACTIVE);
+    Assertions.assertEquals(
+        toggleStatusesService.getTableToggleStatus("dummy2", "db", "tbl").getStatus(),
+        ToggleStatusEnum.ACTIVE);
+    Assertions.assertEquals(
+        toggleStatusesService.getTableToggleStatus("dummy2", "db", "testtbl1").getStatus(),
```

Comment:
how is this test working if service is not calling repository yet?

Reply:
it was failing :) 
Changed that, so right now instead of directly returning in the repository, we will do a exact matching for simplicity.